cartridge/roles/coordinator.lua

```diff
@@ -0,0 +1,293 @@
+local log = require('log')
+local fiber = require('fiber')
+local checks = require('checks')
+local errors = require('errors')
+local netbox = require('net.box')
+local membership = require('membership')
+
+local vars = require('cartridge.vars').new('cartridge.roles.coordinator')
+local topology = require('cartridge.topology')
+local confapplier = require('cartridge.confapplier')
+
+local AppointmentError = errors.new_class('AppointmentError')
+local CoordinatorError = errors.new_class('CoordinatorError')
+local NetboxConnectError = errors.new_class('NetboxConnectError')
+
+vars:new('membership_notification', membership.subscribe())
+vars:new('connect_fiber', nil)
+vars:new('topology_cfg', nil)
+vars:new('conn', nil)
+vars:new('options', {
+    RECONNECT_PERIOD = 5,
+    IMMUNITY_TIMEOUT = 15,
+    NETBOX_CALL_TIMEOUT = 1,
+})
+
+vars:new('healthcheck', function(members, instance_uuid)
```

Comment:
Why not `local healthcheck `? I don't see any mentions in test (it's a single reason I think to store it into vars)

Reply:
It's an extension point. I fear someone would need to monkeypatch it and use another predicate.

Reply:
Ok. Sometimes it's better to write some words in comment to eliminate possible misunderstanding. Thanks for explanation.